.travis.yml

```diff
@@ -1,6 +1,12 @@
 language: ruby
 sudo: required
 
+addons:
+  apt:
+    packages:
+      - unzip
+      - libaio-dev
```

Comment:
The packages listed on `install.sh` L8 are `libaio1` and `rpm`. Why the difference here?